.github/workflows/release.yml

```diff
@@ -54,6 +60,16 @@ jobs:
           sudo apt-get install -y make g++ libssl-dev gcc-aarch64-linux-gnu
           rustup target add aarch64-unknown-linux-gnu
 
+      - name: Install Rust crosscompile tools (musl)
```

Comment:
is cross compilation really a thing here - or do you mean in cross musl vs glibc?